### ISSUE TITLE
[Feature]: Ability to set proot's built in qemu flag

### ISSUE THREAD

**Issue by jsonmona** [termux/proot-distro#418]
### Feature description

proot has flag to run every guest executables with certain emulator. It can help with lack of binfmt_misc.

A flag to pass the argument would be useful.

Although manual says QEMU, it is compatible with any software like binfmt_misc.

```
proot manual:
-q *command*  Execute guest programs through QEMU as specified by *command*
```

**Comment by jsonmona** [comment]
No. I do not want qemu because I have actual aarch64 binaries in the rootfs. `wine` is the only i386 binary I have in my rootfs. I'm trying to use box86 emulator, which requires both i386 library AND aarch64 library to be present.

With that being said, I'll test if `-q` flag indeed works in this scenario. Although box86 fallback to execute natively on non-i386 binary, I'm worried if proot will try to run box86 even on that fallback path, causing an infinite loop.

**Comment by zavocc** [comment]
proot's `-q` argument only accepts qemu, since it passes certain qemu-user parameters anyway when executing a foreign binary inside proot

**Comment by jsonmona** [comment]
Ah, I see. I should've tested things before creating an issue. Thanks you all for clarifying! ❤️
